src/content/docs/teams.mdx

```diff
@@ -72,3 +72,29 @@ However you're welcome to [contact us](mailto:contact@shorebird.dev) and we
 will be happy to help you transfer ownership of an app.
 
 https://github.com/shorebirdtech/shorebird/issues/1225
+
+# FAQ
+
+## Another person purchased Shorebird, but they want me to create the apps?
```

Comment:
Maybe we can also file an issue and link to it (to support being able to create apps directly from the console)?